subsys/net/ip/net_context.c

```diff
@@ -1346,6 +1485,7 @@ NET_CONN_CB(tcp_syn_rcvd)
 	 * If we receive SYN, we send SYN-ACK and go to SYN_RCVD state.
 	 */
 	if (NET_TCP_FLAGS(pkt) == NET_TCP_SYN) {
+		int r;
```

Comment:
very minor nit picking, the code would look better if the "int r" is after the "struct sockaddr" line